src/scripts/lib/db.js

```diff
@@ -259,12 +264,20 @@ export default class Db {
       });
   }
 
-  getLocal (key) {
-    const collection = localStorage.getItem(key);
-    if (!collection) {
-      return null;
+  async getLocal (key) {
+    const localStorageData = localStorage.getItem(key);
```

Comment:
Yeah, it could have returned the stale projects/clients for the first time after update. This method also had another issue with returning unparsed JSON, but it's all fixed up now. Thanks for heads-up! 👍🏻